source/_integrations/speedtestdotnet.markdown

```diff
@@ -43,7 +43,7 @@ monitored_conditions:
       description: "The upload speed (Mbit/s)."
 server_id:
   description: Specify the speed test server to perform the test against.
-  required: false
+  required: true
```

Comment:
This option is not required.

Code reference:
<https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/speedtestdotnet/__init__.py#L26>

Reply:
The code indeed states it's not required. I'll create an issue about this instead. Speedtest.net is somehow not working over here without server_id

Reply:
That is fine, but it still means this change has to be reverted right now.